en/api/action.md

```diff
@@ -36,54 +36,6 @@ Chart actions supported by ECharts are triggered through [dispatchAction](~echar
 
 **Attention: ** The `?:` note in the code shows that this attribute is optional. *EVENT:* stands for the event that triggers action.
 
-## highlight(Action)

```

Comment:
We shouldn't remove them as they are still working for other series or components.

The v5 upgrade guide indicates that the `highlight` action & `downplay` action in the `sunburst` series are deprecated but not in all series.

Perhaps we should add new actions for `sunburstHighlight` & `sunburstUnhighlight`.